src/main/resources/truststore/sni-cloudflaressl-com.pem

```diff
@@ -1,74 +1,94 @@
 -----BEGIN CERTIFICATE-----
-MIIFKjCCBNCgAwIBAgIQBV7cIQB7aqGLU47LI4aV+TAKBggqhkjOPQQDAjBKMQsw

```

Comment:
Unsure if this cert is required anymore, since the W3C validator is gone? Or maybe it's used for something else?

Reply:
Or if it's still required, maybe it should be renamed to clarify what it's actually used for?

Reply:
I'll remove it, pretty sure it was for the w3c-validator.